### ISSUE TITLE
Issue while installing Android Emulator Hypervisor for AMD Processor

### ISSUE THREAD

**Issue by Dinesh-Murugesan** [google/android-emulator-hypervisor-driver-for-amd-processors#25]
I am facing the below issue while trying to install Android Emulator for AMD Processor

"Running Android Emulator Hypervisor Driver for AMD Processors installer
[SC] ControlService FAILED 1062:

The service has not been started.

[SC] DeleteService SUCCESS
[SC] StartService FAILED with error 4294967201.
Android Emulator Hypervisor Driver for AMD Processors installation failed. To install Android Emulator Hypervisor Driver for AMD Processors follow the instructions found at: https://github.com/google/android-emulator-hypervisor-driver-for-amd-processors
Done"

I have enable the virtualization in BIOS and disabled Hyper-V in the windows feature. But still I am not able to install the Android Emulator. Please help

**Comment by wesleydeland** [comment]
I am also having this exact issue. Please advise

**Comment by Taogle2018** [comment]
Can you run "emulator-check.exe hyper-v" and post the result?

**Comment by MagicCrafter200** [comment]
I had the same problem, and if I run "emulator-check.exe hyper v" my computer opens a window, but close it instantly

**Comment by Taogle2018** [comment]
@MagicCrafter200 This is what it looks like when I run it on my desktop.
![Capture](https://user-images.githubusercontent.com/11847425/88702216-e2c34380-d0bf-11ea-8b58-35fd5e252868.PNG)
I don't quite understand what do you mean by "opens a window, but close it instantly".

**Comment by Veteranfighter** [comment]
I have the same problem. When I run emulator-check.exe hyper-v that comes out: 
![Capture](https://user-images.githubusercontent.com/63098858/88851151-a2adaf00-d1ec-11ea-8a3b-dd53b9231214.PNG)

**Comment by Taogle2018** [comment]
@Veteranfighter Did you turn on virtualization in BIOS?

**Comment by danherrera** [comment]
After verifying that none of the virtualization features were enabled and the BIOS virtualization is disabled, I also am encountering this. The `silent_install.bat` simply terminates in the exact error already posted here.

**Comment by tarpaha** [comment]
If it helps - in Windows 10 ver.2004 I can disable Hyper-V only after removing "Hyper-V", "Windows Hypervisor Plarform" **and** "Virtual Machine Platform" components from Windows features.
Until that "emulator-check.exe hyper-v" showed that Hyper-V is enabled.

**Comment by karanjadhav12** [comment]
I am facing the same problem . Virtualization is enabled from bios. Hypervisor platform is unchecked from windows features but when i typed "emulator-check.exe hyper v" in cmd it shows 2 adn hyper v is enabled.
what to do now to install emulator? help

**Comment by Taogle2018** [comment]
Did you have a chance to read the following wiki page?
https://github.com/google/android-emulator-hypervisor-driver-for-amd-processors/wiki/Is-Hyper-V-really-disabled%3F

But be noted that certain version of Windows 10 has a bug. Basically you cannot disable Hyper-V once you enable it. I hope you are not getting into that trouble.

**Comment by karanjadhav12** [comment]
Done. Android Emulator for AMD processors installed. Thank You!!!

**Comment by Taogle2018** [comment]
Great to hear that the problem is solved.

**Comment by jchaitanya16** [comment]
You have to enable the SVM mode to enable in BIOS. Even i have faced the issue then have enabled it worked out. To help out others i have made youtube video so that it can help others who are facing the issue like me. Please checkout to visualize the steps and rectify it. https://youtu.be/OB1IxZ8hq7g

**Comment by linus-ha** [comment]
> 
> 
> Did you have a chance to read the following wiki page?
> https://github.com/google/android-emulator-hypervisor-driver-for-amd-processors/wiki/Is-Hyper-V-really-disabled%3F
> 
> But be noted that certain version of Windows 10 has a bug. Basically you cannot disable Hyper-V once you enable it. I hope you are not getting into that trouble.

So am I seeing this right? You cannot use the android emulator while running WSL / Docker for Desktop? Okay, sorry: Disabling the Windows Virtual Machine Platform is not an option for me. Seems to be a good time switching to Linux.

**Comment by XeonG** [comment]
"Windows Subsystem for Linux version 2 requires Virtual Machine Platform" so if you use WSL2 with that feature on you can't use this?

**Comment by DPSB** [comment]
Yes. But you still could use WSL1 if that is enough. I have disabled vbs and hyper-v and virtual machine platform and so on, and when i run:
`wsl --set-default-version 2`
i get the following output:
```
Please enable the Virtual Machine Platform Windows feature and ensure virtualization is enabled in the BIOS.
For information please visit https://aka.ms/wsl2-install
```
But my debian VM running on WSL1 still is working fine.

**Comment by earonesty** [comment]
[SC] StartService FAILED with error 4294967201 + this error in the event log "The resource loader failed to find MUI file".   SVM definitely on (i used to run an earlier version of android studio just fine), hyperv definitely off (checked with powershell, etc.)

**Comment by DPSB** [comment]
@earonesty 
You might take a look [here](https://github.com/google/android-emulator-hypervisor-driver-for-amd-processors/issues/10), there is some more usefull information related to this error code

Make sure Hyper-V, Guarded Host, Virtual Machine Platform,  Windows Sandbox & Microsoft Defender Application Guard are disabled in the optional features, and check if your antivirus is blocking anything.

**Comment by Gianguyen1234** [comment]
You can try this way, I fix it by turning on virtuallization on Bios:
https://youtu.be/9LVazV4IJ_g
